cli/flox/src/utils/errors.rs

```diff
@@ -312,8 +312,8 @@ pub fn format_core_error(err: &CoreEnvironmentError) -> String {
 
             This is likely due to a corrupt environment.
         "},
-
         CoreEnvironmentError::CreateTempdir(_) => display_chain(err),
+        CoreEnvironmentError::Auth(err) => display_chain(err),
```

Comment:
(unrelated line)
Are you viewing this change as we manually tested the parts we care about, and test coverage/potential rough edges around login will be handled as we revisit/rewrite?

Reply:
Yes. I want to take a step back and take a holistic look at how and where we're doing auth, prompting for login, etc.